themes/default/content/resources/how-to-build-azure-serverless-application/index.md

```diff
@@ -0,0 +1,256 @@
+---
+title: "How do build a serverless applications with Azure, blob storage and API management"
```

Comment:
How to instead of How do?

Did this come from us ... or machine generated?

Reply:
This example is my query and theres just a type of do vs to as this page wasn't fully edited I was just testing some hugo styling I needed to get working, so I grabbed the first work in progress in my list.  question #2 is where does the casing come from, this is from the PulumiAI questions, in this case users are unlikely to ask their questions in proper casing, thus the replies also are not. One of many little content challenges we'll need to address to make something like this scale.